server/controllers/search/lib/types.coffee

```diff
@@ -1,13 +1,18 @@
 CONFIG = require 'config'
 
-entitiesDbName = CONFIG.db.name 'entities'
+localAndRemoteEntity = (type)-> { indexes: [ 'wikidata', CONFIG.db.name('entities') ], type }
+remoteOnlyEntity = (type)-> { indexes: [ 'wikidata' ], type }
 
 typesData =
-  works: { indexes: [ 'wikidata', entitiesDbName ], types: ['works', 'entity'] }
-  humans: { indexes: [ 'wikidata', entitiesDbName ], types: ['humans', 'entity'] }
-  series: { indexes: [ 'wikidata', entitiesDbName ], types: ['series', 'entity'] }
-  users: { indexes: [ CONFIG.db.name('users') ], types: ['user'] }
-  groups: { indexes: [ CONFIG.db.name('groups') ], types: ['group'] }
+  works: localAndRemoteEntity 'works'
+  humans: localAndRemoteEntity 'humans'
+  series: localAndRemoteEntity 'series'
+  genres: remoteOnlyEntity 'genres'
+  movements: remoteOnlyEntity 'movements'
+  publishers: remoteOnlyEntity 'publishers'
+  collections: remoteOnlyEntity 'collections'
+  users: { indexes: [ CONFIG.db.name('users') ], type: 'user' }
```

Comment:
maybe push the logic with a `localOnlyEntity`

Reply:
done in fd34cf5